files/mysql-setup.sql

```diff
@@ -0,0 +1,7 @@
+SET tx_isolation = 'READ-COMMITTED';
```

Comment:
I think this file may be better contributed to the `orca-sql-mysql` module.

Reply:
Sorry, I don't know what this means. Where is this module?

Reply:
Oops, right... https://github.com/spinnaker/orca/tree/master/orca-sql-mysql

Reply:
fixed up, https://github.com/spinnaker/orca/pull/2627